x/auth/keeper/keeper.go

```diff
@@ -102,6 +102,8 @@ type AccountKeeper struct {
 	Schema collections.Schema
 	Params collections.Item[types.Params]
 
+	// only use for upgrade handler
+	accountNumber collections.Sequence
```

Comment:
can we mark this as deprecated if its not used any longer

Reply:
updated